lib/bloc/account/account_bloc.dart

```diff
@@ -165,13 +167,22 @@ class AccountBloc extends Cubit<AccountState> with HydratedMixin {
     _log.v(
         "lnurlWithdraw amount: $amountSats, description: '$description', reqData: $reqData");
     try {
-      return _breezLib.lnurlWithdraw(
+      final callbackStatus = await _breezLib.lnurlWithdraw(
         amountSats: amountSats,
         reqData: reqData,
         description: description,
       );
+      if (callbackStatus is sdk.LnUrlWithdrawCallbackStatus_Ok) {
+        _paymentResultStreamController.add(
+          PaymentResult(
```

Comment:
@erdemyerebasmaz  We have "InvoicePaid" event for incoming payments and we want to add also "PaymentSent" event as you suggested but it will not fire for incoming payments....

Reply:
We can use LnUrlWithdrawCallbackStatus for lnurl withdraw.